parser.go

```diff
@@ -629,7 +629,9 @@ func parseNumberMenu(input string) (intRanges, intRanges, stringSet, stringSet)
 	otherInclude := make(stringSet)
 	otherExclude := make(stringSet)
 
-	words := strings.Fields(input)
+	words := strings.FieldsFunc(input, func(c rune) bool {
+		return strings.ContainsRune(" \t,", c)
```

Comment:
Why not?

```go
return unicode.IsSpace(c) || c == ','
```

Reply:
1.  `import "unicode"` adding: https://github.com/Jguer/yay/pull/329#issuecomment-378956359; oh, there was `regexp`; oh, `unicode` doesn't increase the binary size, OK.
2. I wanted to make it simpler.

Reply:
Changed.

Reply:
That graph always scares me, but I guess if you want clean graphs, you rewrite everything yourself and after that's done port to C for performance sake :laughing:. 
I agree that it's simpler to read.

Reply:
Big graph like this is normal for stdlib dependencies, not for third-party. But looks scary, yes.